dashboard/config/scripts/allthethings.script

```diff
@@ -1,5 +1,6 @@
 hideable_lessons true
 lesson_extras_available true
+published_state 'beta'
```

Comment:
intended change? (same question for the reorder below)

Reply:
Huh I actually don't know. I edited the script through the UI and saved it - does this mean I should have seeded first?

Reply:
Edit: I went back and seeded first and got similar changes.

Reply:
Asked in #dashboard-dev and this was the response:
"these are mostly harmless -- when we update the format that these files get serialized into, we don't always go back and update all the files to match. however for your own sanity what I'd recommend is including only the bare minimum you need to accomplish your change, to minimize merge conflicts. in this case, only the .script file does anything, so you can revert the entire .script_json file."

^I'll go ahead and take that recommendation to remove the unneeded changes